doc/sphinx/azhelpgen/azhelpgen.py

```diff
@@ -10,10 +10,10 @@
 from sphinx.util.compat import Directive
 from sphinx.util.nodes import nested_parse_with_titles
 
-from azure.cli.application import Application, Configuration
+from azure.cli.application import APPLICATION, Configuration
 import azure.cli._help as _help
 
-app = Application(Configuration([]))
+app = APPLICATION
```

Comment:
Yes, this looks similar to the kind of change I had to make in 'dump_command_table.py'. Can you pretty please just verify that 'python scripts\dump_command_table --params ids' still works after these changes. I can't see why it wouldn't, but stranger things have happened.